e2e/pages/adf/contentServicesPage.ts

```diff
@@ -66,20 +66,91 @@ export class ContentServicesPage {
     copyButton = element(by.css('button[data-automation-id="content-node-selector-actions-choose"]'));
     searchInputElement = element(by.css('input[data-automation-id="content-node-selector-search-input"]'));
     shareNodeButton = element(by.cssContainingText('mat-icon', ' share '));
+    nameColumnHeader = 'name';
+    createdByColumnHeader = 'createdByUser.displayName';
+    createdColumnHeader = 'createdAt';
+    deleteContentElement = element(by.css('button[data-automation-id*="DELETE"]'));
+    metadataAction = element(by.css('button[data-automation-id*="METADATA"]'));
+    versionManagerAction = element(by.css('button[data-automation-id*="VERSIONS"]'));
+    moveContentElement = element(by.css('button[data-automation-id*="MOVE"]'));
+    copyContentElement = element(by.css('button[data-automation-id*="COPY"]'));
+    lockContentElement = element(by.css('button[data-automation-id="DOCUMENT_LIST.ACTIONS.LOCK"]'));
+    downloadContent = element(by.css('button[data-automation-id*="DOWNLOAD"]'));
+
+    checkLockedIcon(content) {
+        let lockIcon = element(by.cssContainingText('div[filename="' + content + '"] mat-icon', 'lock'));
```

Comment:
I think the first part of the locator, the div, can be obtained using datatable PO methods.